vignettes/DirichletMultinomial.Rmd

```diff
@@ -0,0 +1,320 @@
+---
+title: "DirichletMultinomial for Clustering and
+        Classification of Microbiome Data"
+author:
+- name: "Khadijah Amusat"
+  affiliation: "Vignette translation from Sweave to R Markdown / HTML"
+date: "`r format(Sys.time(), '%B %d , %Y')`"
+output:
+    BiocStyle::html_document:
+    number_sections: true
+    toc: true
+    toc_depth: 4
+bibliography: references.bib
+link-citations: TRUE
+package: DirichletMultinomial    
+vignette: >
+  %\VignetteIndexEntry{DirichletMultinomial for Clustering and
+                       Classification of Microbiome Data}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r style, echo = FALSE, results = 'asis'}
+  BiocStyle::markdown()
+```
+
+This document illustrates the main features of the `r
+Biocpkg("DirichletMultinomial")`, and in the process replicates key tables and
+figures from [@10.1371/journal.pone.0030126].
+We start by loading the package, in addition to the packages *lattice* (for
+visualization) and *parallel* (for use of multiple cores during
```

Comment:
`lattice` is CRAN package, so we should generate the link with

    `r CRANpkg("lattice")`

And I believe `parallel` is in base R so you can use

    `r Rpackage("parallel")`